server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -367,21 +357,17 @@ export class MongoDbFactory implements core.IDbFactory {
                  when trying to connect to global db.`);
         // Need to cast to any before MongoClientOptions due to missing properties in d.ts
         const options: MongoClientOptions = {
-            autoReconnect: true,
-            bufferMaxEntries: this.bufferMaxEntries ?? 50,
             keepAlive: true,
             keepAliveInitialDelay: 180000,
-            reconnectInterval: 1000,
-            reconnectTries: 100,
             socketTimeoutMS: 120000,
-            useNewUrlParser: true,
```

Comment:
Do you have any articles talking about how to connect to mongodb and how to reconnect on network blips?